### PR TITLE
Changing logo of AniNews and Adding ResetEra

### PR DESCRIPTION
**Pull request by frozaum**
# AniNews
- Well, they just changed the logo, nothing else.

# ResetEra
Forum list
Thread
Preferences

### CODE REVIEW

File: ResetEra/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "name": "fror",
+      "id": "515668127829458945"
+    },
+    "service": "ResetEra",
+    "description": {
+      "en": "ResetEra is the internet's premier video gaming forum destination. A savory assortment of enthusiasts, journalists, & developers."
+    },
+    "url": [
+      "www.resetera.com"
+    ],
```
Comment (Bas950) [inline ResetEra/dist/metadata.json]: url shouldnt be in an array if its only 1

File: AniNews/dist/metadata.json [hunk 2]
```diff
@@ -19,8 +19,8 @@
   "url": [
     "www.aninews.live"
```
Comment (Alanexei) [inline AniNews/dist/metadata.json]: Should be a string if there's only one.

Comment (frozaum) [reply AniNews/dist/metadata.json]: fuck off

File: ResetEra/dist/metadata.json (hunk 1 above)
Comment (frozaum) [reply ResetEra/dist/metadata.json]: okay

**Pull request merged**